src/ut_kvp.c

```diff
@@ -596,11 +632,11 @@ static ut_kvp_instance_internal_t *validateInstance(ut_kvp_instance_t *pInstance
 
 static bool str_to_bool(const char *string)
 {
-    if ( strcasecmp( string, "true" ) == 0 )
+    if (strcasecmp(string, "true") == 0)
     {
         return true;
     }
-    if ( strcasecmp( string, "false" ) == 0 )
+    if (strcasecmp(string, "false") == 0)
```

Comment:
just remove the false check for string.

If !true, then all other cases are false, no checking required.